src/NexusMods.App.UI/Pages/ModLibrary/FileOrigins/FileOriginEntry/FileOriginEntryViewModel.cs

```diff
@@ -20,7 +23,11 @@ public class FileOriginEntryViewModel : AViewModel<IFileOriginEntryViewModel>, I
     public string Version { get; }
     public Size Size { get; }
     public DateTime ArchiveDate { get; }
-    public ReactiveCommand<Unit, Unit> AddToLoadoutCommand { get; init; }
+    public ReactiveCommand<IModInstaller?, Unit> AddToLoadoutCommand { get; init; }
```

Comment:
Don't pass IModInstaller as a parameter, this is never passed from UI, this is a hack to reuse the command for multi selection. This just makes the code confusing. 
You can use a method if you really want to reuse code.